shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/transferobject/LocationTO.java

```diff
@@ -80,4 +81,20 @@ record FacilityLocationTO(
     @Builder(toBuilder = true)
     public FacilityLocationTO { }
   }
+
+
+  record GeoLocationTO(
```

Comment:
you would also need to update LocationTODeserializer, LocationService and LocationMapper

Reply:
Fixed.